site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -4,18 +4,164 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
         beforeEach(() => {
             cy.login('instructor');
             cy.visit(['sample', 'config']);
+            checkCheckbox('[data-testid="display-rainbow-grades-summary"]');
+            cy.visit(['sample', 'reports', 'rainbow_grades_customization']);
+            reset();
```

Comment:
I like how you are resetting the values before each instead of after each, that is 'best practice'.